src/framework/COScript.m

```diff
@@ -405,6 +405,56 @@ - (void)deleteObjectWithName:(NSString*)name {
     [_mochaRuntime removeObjectWithName:name];
 }
 
+# pragma mark - require
+
+- (NSURL*)resolveModule:(NSString *)module currentURL:(NSURL*)currentURL {
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    BOOL isRelative = [module characterAtIndex:0] == '.';
```

Comment:
I think you're ok by how unicode works, but maybe `hasPrefix: @"."]` would be safer.